### ISSUE TITLE
same thing but for chrome?

### ISSUE THREAD

**Issue by dragonGRaf** [eric-bixby/auto-sort-bookmarks-webext#117]
is there a copy for chrome? i tired Foxified but it had a error
![image](https://github.com/user-attachments/assets/1b1362ca-7f60-480d-8cfa-1922f5e63d96)

**Comment by eric-bixby** [comment]
https://foxified.org

Probably won't work.  This extension needs to access the bookmarks and storage APIs, which I'm guessing are either blocked by the sandbox or not supported by Foxified.

Anyway, I will consider supporting Chrome (just not sure when).

**Comment by eric-bixby** [comment]
Duplicate of #38
